tests/python/relay/test_pass_partition_graph.py

```diff
@@ -973,6 +951,93 @@ def test_exec(mod, params, ref_mod, ref_params, out_shape):
     # test_exec(mod, params, ref_mod, ref_params, (1, 1000))
 
 
+def test_multiple_use_of_an_output():
```

Comment:
need to add this function to the main. It is not being run

Reply:
It would be still executed with pytest. But I added to main as well.